src/arviz_plots/backend/matplotlib/__init__.py

```diff
@@ -119,6 +119,37 @@ def get_default_aes(aes_key, n, kwargs=None):
     return get_agnostic_default_aes(aes_key, n, kwargs)
 
 
+def dealiase_aes_value(aes, value):
+    try:
+        if value.startswith(f"{aes}_"):
+            index = int(value.rsplit("_")[-1])
+        else:
+            index = int(value[1:])
+        dealiased_value = get_default_aes(aes, index+1)[index]
```

Comment:
add check about unique elements and error out if `index` is larger than that.

Reply:
I think this is not necessary, I may have said before that this was necessary. But the matplotlib syntax for "CN" allows N to be larger than the length of the cycler. So C100 is a valid syntax.